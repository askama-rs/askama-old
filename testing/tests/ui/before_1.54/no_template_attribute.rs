../since_1.54/no_template_attribute.rs