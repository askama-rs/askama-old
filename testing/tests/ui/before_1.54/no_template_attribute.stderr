error: no attribute 'template' found
 --> tests/ui/before_1.54/no_template_attribute.rs:3:10
  |
3 | #[derive(Template)]
  |          ^^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
